### DISCUSSION TITLE
How can i add contextMenus events?

### DISCUSSION THREAD

**Discussion by frankye23** [antfu/vitesse-webext#141] [Q&A]
after i add contextMenus  into 'permissions' array in manifest, what should i do next?

**Comment by d-bucur** [comment]
There are different types of contexts you can add them to, [here](https://stackoverflow.com/questions/19468429/add-contextmenu-items-to-a-chrome-extensions-browser-action-button/26760703#26760703) is an example using 'browser_action' (when you right click on the action button)
